lib/rspec/page-regression/matcher.rb

```diff
@@ -7,7 +7,14 @@ module RSpec::PageRegression
       args ||= {}
       verify_arguments(args)
       @responsive_filepaths = FilePaths.responsive_file_paths(RSpec.current_example, args)
-      Renderer.render_responsive(page, @responsive_filepaths)
+
+      if args.key?(:selector)
+        opt = { selector: args[:selector] }
+      else
+        opt = { full: args.fetch(:full, true) }
+      end
```

Comment:
Can we remove the `else` part here? `renderer.rb` already sets `options[:full]`.
https://github.com/teachbase/rspec-page-regression/blob/match-part-page/lib/rspec/page-regression/renderer.rb#L13